### DISCUSSION TITLE
how do i remove this?

### DISCUSSION THREAD

**Discussion by prkupin** [keiwando/toonsquid#685] [Q&A]
![IMG_0515](https://github.com/user-attachments/assets/8e49ad18-cdbd-420f-bbf5-6cb3c50d0e12)
this weird time thing happened while i was animating, and i don’t know how to remove or disable it. i was wondering if anyone has gone through something similar?

**Comment by keiwando** [comment]
Hi!
The timeline got zoomed out really far. You can pinch with two pingers to zoom it back in. 
Apparently, sometimes the iPad reports to the app that the user pinched to zoom out extremely far. It might be an issue with palm rejection not working for a split second, but I'm not too sure, since this is a very rare occurrence. The upcoming 2.0 update will contain a fix to ignore such unrealistic sudden zoom changes, but for now, you just have to manually zoom back in. Sorry about that!
